src/v2/Apps/ViewingRoom/Components/ViewingRoomsFeaturedRail.tsx

```diff
@@ -23,18 +24,15 @@ export const ViewingRoomsFeaturedRail: React.FC<ViewingRoomsFeaturedRailProps> =
   }
 
   const carouselItemRender = (
-    {
-      heroImageURL,
-      slug,
-      title,
-      partner,
-      status,
-      distanceToOpen,
-      distanceToClose,
-    },
+    { image, slug, title, partner, status, distanceToOpen, distanceToClose },
     slideIndex: number
   ): React.ReactElement => {
     const tag = getTagProps(status, distanceToOpen, distanceToClose)
+    const heroImageURL = resize(image?.imageURLs?.normalized, {
+      height: 370,
```

Comment:
Does this take landscape images into account?

Also wondering  - if these are the display dimensions, should we request a resized image at twice the size to account for retina / higher pixel density displays?

Reply:
oop had the same thought but did not see you'd already commented!